cypress/integration/console/devices/onboarding/qr-scan.spec.js

```diff
@@ -0,0 +1,231 @@
+// Copyright © 2021 The Things Network Foundation, The Things Industries B.V.
```

Comment:
```suggestion
// Copyright © 2022 The Things Network Foundation, The Things Industries B.V.
```
Please use a snippet or some kind of check, otherwise this will pop up again and again.